intel_extension_for_transformers/llm/runtime/graph/fused_attention.md

```diff
@@ -0,0 +1,232 @@
+========================================
+=== GRAPH Profiling ===
+perf_total_per_op_us[                     ADD] =   0.324 ms
+perf_total_per_op_us[                     MUL] =   0.402 ms
+perf_total_per_op_us[                RMS_NORM] =   1.321 ms
+perf_total_per_op_us[                 MUL_MAT] =  10.624 ms
+perf_total_per_op_us[                   SCALE] =   0.954 ms
+perf_total_per_op_us[                     CPY] =   3.104 ms
+perf_total_per_op_us[                 RESHAPE] =   0.425 ms
+perf_total_per_op_us[                    VIEW] =   0.748 ms
+perf_total_per_op_us[                 PERMUTE] =   0.316 ms
+perf_total_per_op_us[               TRANSPOSE] =   0.102 ms
+perf_total_per_op_us[                GET_ROWS] =   0.021 ms
+perf_total_per_op_us[           DIAG_MASK_INF] =   0.111 ms
+perf_total_per_op_us[                SOFT_MAX] =   1.362 ms
+perf_total_per_op_us[                    ROPE] =   1.874 ms
+perf_total_per_op_us[                 MUL_QKV] =   6.001 ms
+perf_total_per_op_us[                FFN_SILU] =  14.542 ms
+perf_total_per_op_us[           INNER PRODUCT] =   3.314 ms
+========================================
+=== GRAPH Profiling ===
+perf_total_per_op_us[                     ADD] =   0.354 ms
+perf_total_per_op_us[                     MUL] =   0.391 ms
+perf_total_per_op_us[                RMS_NORM] =   1.379 ms
+perf_total_per_op_us[                 MUL_MAT] =  10.610 ms
+perf_total_per_op_us[                   SCALE] =   0.964 ms
+perf_total_per_op_us[                     CPY] =   3.115 ms
+perf_total_per_op_us[                 RESHAPE] =   0.430 ms
+perf_total_per_op_us[                    VIEW] =   0.866 ms
+perf_total_per_op_us[                 PERMUTE] =   0.336 ms
+perf_total_per_op_us[               TRANSPOSE] =   0.109 ms
+perf_total_per_op_us[                GET_ROWS] =   0.022 ms
+perf_total_per_op_us[           DIAG_MASK_INF] =   0.108 ms
+perf_total_per_op_us[                SOFT_MAX] =   1.410 ms
+perf_total_per_op_us[                    ROPE] =   1.959 ms
+perf_total_per_op_us[                 MUL_QKV] =   5.826 ms
+perf_total_per_op_us[                FFN_SILU] =  14.737 ms
+perf_total_per_op_us[           INNER PRODUCT] =   3.378 ms
+========================================
+
+model_print_timings:        load time = 10987.95 ms
+model_print_timings:      sample time =     2.38 ms /     4 runs   (    0.60 ms per token)
+model_print_timings: prompt eval time =  9748.26 ms /  1975 tokens (    4.94 ms per token)
+model_print_timings:        eval time =   150.93 ms /     3 runs   (   50.31 ms per token)
+model_print_timings:       total time = 11175.74 ms
+========== eval time log of each prediction ==========
+prediction   0, time: 9748.26ms
+prediction   1, time: 50.56ms
+prediction   2, time: 50.00ms
+prediction   3, time: 50.37ms
+```
+
+Where:
+|                  | pred 0 fused attn disabled | pred 0 fused attn enabled | pred 3 fused attn disabled | pred 3 fused attn enabled |
```

Comment:
users don't know what's pred0, you can use 1st token

Reply:
Updated in c4179aa